book/modules/appendix1.tex

```diff
@@ -351,24 +388,189 @@
+		\begin{align*}
+			{\footnotesize
+			\systeme[abc]{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				2&7&5 & 2\\
+				-1&-4&0 & 11
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-2\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				-1&-4&0 & 11
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3+\text{row}_1+\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				0&0&3 & 12
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme[abc]{
+				a+3b+2c=1,
+				b+c=0,
+				3c=12
+				}
+			}
+		\end{align*}
+		The third row reveals that $c=4$ and we substitute that into the second row to find $b=-4$. Now we can substitute $b=-4$ and $c=4$ into the first row and we obtain $a=5$.
+		
+		As a vector, the solution is $\mat{5\\-4\\4}$ and since it is unique, the solution set is $\Set*{\mat{5\\-4\\4}}$.
 	\end{example}
 
 	\begin{example}
-		XXX Finish. Unique solution to a 3x4 system that needs a row swap; Use back subsitution
+		Consider the system
+		\[
+			\systeme[tsr]{
+				3t+s+13r=-2,
+				t+5r=1,
+				-t+s-6r=-8
+			}.
+		\]
+		Again, we row reduce the corresponding augmented matrix to find an equivalent system\index{System of linear equations!equivalent} that is easier to find the solution set.
+		\begin{align*}
+			{\footnotesize
+			\systeme[tsr]{
+				3t+s+13r=-2,
+				t+5r=1,
+				-t+s-6r=-8
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				3&1&13 & -2\\
+				1&0&5 & 1\\
+				-1&1&-6 & -8
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_1\leftrightarrow\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				3&1&13 & -2\\
+				-1&1&-6 & -8
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-3\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				-1&1&-6 & -8
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3+\text{row}_1-\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&5 & 1\\
+				0&1&-2 & -5\\
+				0&0&1 & -2
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme[tsr]{
+				t+5r=1,
+				s-2r=-5,
+				r=-2
+				}
+			}
+		\end{align*}
+		Our equivalent system reveals $r=-2$ which we can substitute back into the first and second rows to find that $t=11$ and $s=-9$, respectively.
+		
+		The solution vector is $\mat{11\\-9\\-2}$ and so the solution set is $\Set*{\mat{11\\-9\\-2}}$.
 	\end{example}
 
 	In the examples so far, we've stopped row reducing when the equations became simple enough
 	to solve by inspection. However, we could continue row reducing until the system is as simple as possible.
 
 	\begin{example}
-		XXX Finish. Example 1 again, but this time put it in rref. 
+		We return to the earlier system
+		\[
+			\systeme[abc]{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}
+		\]
+		except now let's try row reducing as much as possible.
```

Comment:
The first paragraph of an example should always specify the question. In this case, "Solve the system..."

In the next paragraph, "Notice, that this system was solved using a combination of row reduction and substution in a previous example. This time, let us use only row reduction..."  You can cite the work from the previous example and use that as a starting point. Don't reduce from the start again.